### PR TITLE
Add ability to override configuration on test pages.

### PR DESCRIPTION
**Pull request by mtutynina**

### CODE REVIEW

File: js/appInit.ts [hunk 1]
```diff
@@ -42,6 +42,11 @@ function registerTableView() {
     window["ActionBlock"] = ActionBlock;
 }
 
+/** Override configuration on test pages */
+function overrideConfig() {
+    overrideConfiguration(window["pokerConfig"] || {});
```
Comment (kant2002) [inline js/appInit.ts]: Are you able export `overrideConfiguration` and not create custom function `overrideConfig()`

Comment (mtutynina) [reply js/appInit.ts]: I can't pass parameter to this function from test project.

Comment (kant2002) [reply js/appInit.ts]: What prevent passing parameter? Since you export function, it should just work.

**Review by kant2002 — APPROVED** [review]
LGTM

**Pull request merged**